src/components/Navigation.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+const Navigation = () => {
+  const [listFontWeight, setListFontWeight] = useState('normal');
+  const [addFontWeight, setAddFontWeight] = useState('normal');
+
+  const toggleBold = (event) => {
+    if (event.target.className === 'list-link') {
+      setListFontWeight('bold');
+      setAddFontWeight('normal');
+    }
+    if (event.target.className === 'add-link') {
+      setAddFontWeight('bold');
+      setListFontWeight('normal');
+    }
+  };
+  return (
+    <nav>
+      <Link
+        to="/list"
+        className="list-link"
+        onClick={toggleBold}
+        style={{ fontWeight: listFontWeight }}
+      >
+        List
+      </Link>
```

Comment:
I'm going to make a suggestion because I think it will make your lives easier. : )

`NavLink` from react-router has a built-in `activeClassName` for when the `to` prop matches the url. That way, if you are on the `/list` route and you refresh the page, that link will still be the `active` link, and you can give it the bold font and background color based on the `activeClassName`.

https://reactrouter.com/web/api/NavLink

Reply:
I've made changes locally and want to review with Dee before pushing to the remote. Thanks for the tip! It simplifies things a lot.

Reply:
I just learned something new, thanks

Reply:
I learned about `NavLink` from a former Collabie!